web/blog/tenzir-node-v4.25/index.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "Tenzir Node v4.25: Sinks Galore!"
+slug: tenzir-node-v4.25
+authors: [lava]
+date: 2025-01-06
+tags: [release, node]
+comments: true
+---
+
+# Tenzir Node v4.25: Sinks Galore!
+
+Tenzir Node v4.25 adds new sinks for Snowflake, OpenSearch, and Elasticsearch, allowing seamless data integration and output. It also introduces major enhancements to TQL2, including new language features and operator improvements.
+
+![Tenzir Node v4.25](tenzir-node-v4.25.excalidraw.svg)
+
+[github-release]: https://github.com/tenzir/tenzir/releases/tag/v4.25.0
+
+<!-- truncate -->
+
+## New Sinks: Snowflake, OpenSearch, and Elasticsearch
+
+Tenzir Node v4.25 offers several new sinks for sending your data. The [`to_snowflake`](/next/tql2/operators/to_snowflake) sink enables seamless writing of data into
+[Snowflake](https://www.snowflake.com) databases, helping users integrate Tenzir with one of the most popular cloud data platforms. Additionally, the
+new [`to_opensearch`](/next/tql2/operators/to_opensearch) and [`to_elasticsearch`](/next/tql2/operators/to_elasticsearch) sink operators allow direct data output to
+[OpenSearch](https://opensearch.org/) and [Elasticsearch](https://www.elastic.co/elasticsearch), two powerful search and analytics engines.
+
+For more information on using Tenzir in combination with these new sinks, check out our integration pages for [OpenSearch](/next/integrations/opensearch),
+[ElasticSearch](/next/integrations/elasticsearch) and [Snowflake](/next/integrations/snowflake)
```

Comment:
Do we normally link to the `next` version of docs?

Reply:
Yes, because otherwise the Docusaurus link check will fail. It's a framework deficiency.

Reply:
We have to for pages that are only introduced within the same release, otherwise the CI will fail because there are no versioned links for these yet.